test/gateway/eip20_gateway/activate_gateway.js

```diff
@@ -0,0 +1,92 @@
+const Gateway = artifacts.require("./EIP20Gateway.sol")
+    , BN = require('bn.js');
+
+const Utils = require('../../../test/test_lib/utils');
+
+
+contract('GatewayBase.sol', function (accounts) {
+
+    describe('Deactivate gateway', async () => {
```

Comment:
Same as above. Plus: wrong name `GatewayBase`. Plus: deactivate should be in a different file that tests the deactivate method.

Reply:
All the feedback around `activate_gateway.js` is done in pr #518 
Please check [this](https://github.com/OpenSTFoundation/mosaic-contracts/pull/518/commits/51e49539049b83be83e5156fb1ab228da580da94) commit.